.gitpod.yml

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements. See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership. The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# https://www.gitpod.io
+# https://www.gitpod.io/docs/41_Config_Gitpod_File/
+# https://github.com/gitpod-io/workspace-images/tree/master/mysql
+
+image: gitpod/workspace-mysql
+ports:
+- port: 8080
```

Comment:
@svenefftinge I noticed you had used port 8005, what's that for? Just an arbitrary example, or something GitPod requires? How to specify several ports here, like 8080 and 8443? https://www.gitpod.io/docs/43_Config_Ports/ only mentions ranges.

Reply:
Talking about ports... how do you suppress messages like these in GitPod:

> A service is listening on port 8005 but is not exposed
> A service is listening on port 3306 but is not exposed

I suspect these are perhaps ports that the Gradle build opens, but which it doesn't need to expose - nor ask about every time, that's confusing / noisy.

Reply:
@svenefftinge oh I'm dumb, I think I just figured (both of) these out, see 723ca56c8b754a19cda8f202d8d13f04715d6bc5

Reply:
Yes, that's how you do it :)

Reply:
What version of MySQL is this?
Could not get alot of info here: https://hub.docker.com/r/gitpod/workspace-mysql

Reply:
```
$ mysqld --version
mysqld  Ver 5.7.26-0ubuntu0.18.10.1 for Linux on x86_64 ((Ubuntu))
```

Find the `Dockerfile` here: https://github.com/gitpod-io/workspace-images/tree/master/mysql

The mysql-image inherits from [workspace-full](https://github.com/gitpod-io/workspace-images/blob/master/full/Dockerfile) which again inherits from `ubuntu:consmic`. 

The mysql-version is the latest available in `ubuntu:consmic`. Let me know if anther version would be better for some reason :)

Reply:
Hmm,,, MySQL 5.7 is fine and supported.. I believe.